Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js

```diff
@@ -1408,12 +1408,45 @@ Ext.define("Zenoss.InfraDetailNav", {
     }
 });
 
+var truncate = function(cmp_name, width, uid) {
+    var path = uid.replace('/zport/dmd/', ''),
```

Comment:
In 4.x, where the full path was visible, the path started with a `/`.  You can see this in the screenshots shown in the ticket.

To preserve this, the `replace` should be rewritten as `uid.replace('/zport/dmd', '')`.